_posts/2017/2017-11-01-internal-workshops.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: post
+title: "Gradle Workshop"
+date: "2017-11-01 22:35:13"
+image: blog-header/workshop.jpg
```

Comment:
Please attach this image. I've resized it already to our needs.
![workshop](https://user-images.githubusercontent.com/10460958/33425819-2fa29dc0-d5c0-11e7-9d5d-a0f12f448ba5.jpg)